spec/models/bishop_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Bishop, type: :model do
+  describe '#move_legal?' do
+    let(:bishop){Bishop.create(row: 3, col: 3)}
+
+    it "is legal" do
+      expect(bishop.move_legal?(4,4)).to be true

```

Comment:
How does this test involve a king's legal move?  I'm not sure why moving it to another square matters...